### ISSUE TITLE
Can't open Realm in tests.

### ISSUE THREAD

**Issue by htetlynnhtun** [realm/realm-dart#1711]
### What happened?

Failed to run test with the following Realm error
```
00:01 +0 -1: Init [E]
  Realm error : Could not open librealm_dart.dylib. Tried:
  - "/Users/username/.dev-tools/flutter/bin/cache/artifacts/engine/darwin-x64/../Frameworks/librealm_dart.dylib"
  Hint: Did you forget to add a dependency on the realm package?
  package:realm_dart/src/init.dart 71:5                _openRealmLib.throwError
  package:realm_dart/src/init.dart 84:29               _openRealmLib
  package:realm_dart/src/init.dart 120:24              initRealm
  package:realm_dart/src/native/realm_core.dart 47:31  _realmLib.<fn>
  package:realm_dart/src/native/realm_core.dart 55:2   _realmLib
  package:realm_dart/src/native/realm_core.dart        _realmLib
  package:realm_dart/src/native/realm_core.dart 104:5  new _RealmCore
  package:realm_dart/src/native/realm_core.dart 81:24  realmCore
  package:realm_dart/src/native/realm_core.dart        realmCore
  package:realm_dart/src/configuration.dart 85:14      Configuration.defaultStoragePath
  package:realm_dart/src/configuration.dart 95:47      Configuration.defaultRealmPath
  package:realm_dart/src/configuration.dart            Configuration.defaultRealmPath
  package:realm_dart/src/configuration.dart 109:44     Configuration._defaultPath
  package:realm_dart/src/configuration.dart 105:50     new Configuration._
  package:realm_dart/src/configuration.dart 411:14     new InMemoryConfiguration._
  package:realm_dart/src/configuration.dart 174:29     Configuration.inMemory
  test/realm_test.dart 6:39                            main.<fn>
```

### Repro steps

1. Clone [this](https://github.com/htetlynnhtun/realm_test) repo.
2. run `flutter pub get`
3. run `dart run realm install -t macos` (or even `dart run realm_dart install -t macos`
4. run `flutter test test/realm_test.dart

### Version

3.22.1

### What Atlas Services are you using?

Local Database only

### What type of application is this?

Flutter Application

### Client OS and version

macos 14.5

**Comment by nielsenko** [comment]
@htetlynnhtun Thanks for trying realm. This is a duplicate of #1619 so I'll close this one.

**Comment by htetlynnhtun** [comment]
So the fix #1623 is not released for production yet? Right?

**Comment by nielsenko** [comment]
@htetlynnhtun No. It is merged to main, but not released yet.
